steps/data_scraping_steps/scrape_mind_data/scrape_mind_data_step.py

```diff
@@ -30,12 +30,33 @@
     ],
 }
 
+URLS_TO_DISCARD = [
```

Comment:
Possible improvement: could these be a parameter in the config? You can shorten them to just the sub-page for the `information-support/`, with `https://www.mind.org.uk/information-support/` being a static variable?

Reply:
Yep, that's fair - mind adding a ticket to capture this?